docs/getting-started/install-configure.md

```diff
@@ -0,0 +1,299 @@
+---
+title: Install & Configure
+toc: true
+weight: 1
+indent: true
+---
+
+# Install & Configure Quickstart
+
+Crossplane can be easily installed into any existing Kubernetes cluster using
+the regularly published Helm chart. The Helm chart contains all the custom
+resources and controllers needed to deploy and configure Crossplane.
+
+See [Install] and [Configure] docs for installing alternate versions and more
+detailed instructions.
+
+## Get a Kubernetes Cluster
+
+<ul class="nav nav-tabs">
+<li class="active"><a href="#setup-mac-brew" data-toggle="tab">macOS via Homebrew</a></li>
+<li><a href="#setup-mac-linux" data-toggle="tab">macOS / Linux</a></li>
+<li><a href="#setup-windows" data-toggle="tab">Windows</a></li>
+</ul>
+<br>
+<div class="tab-content">
+<div class="tab-pane fade in active" id="setup-mac-brew" markdown="1">
+For macOS via Homebrew use the following:
+
+```
+brew upgrade
+brew install kind
+brew install kubectl
+brew install helm
+
+kind create cluster --image kindest/node:v1.17.5 --wait 5m
```

Comment:
We've reworked the OAM addon such that it should (still validating) only need 1.16, because most cloud providers don't yet support 1.17 per https://github.com/crossplane/addon-oam-kubernetes-local/issues/30.

Reply:
sounds good, happy to adjust down to 1.16 after you've validated all is well

Reply:
- aks supports 1.16.7
- gke supports 1.16.8
- eks supports 1.16.8